rootex/core/renderer/post_processor.cpp

```diff
@@ -153,6 +153,55 @@ class MonochromePostProcess : public PostProcess
 	}
 };
 
+Variant CustomPostProcess::loadRTVAndSRV(const Event* event)
+{
+	RenderingDevice::GetSingleton()->createRTVAndSRV(m_CacheRTV, m_CacheSRV);
+	return true;
+}
+
+CustomPostProcess::CustomPostProcess(String path)
+{
+	postProcessPath = path;
+
+	m_Binder.bind(RootexEvents::WindowResized, this, &CustomPostProcess::loadRTVAndSRV);
+
+	loadRTVAndSRV(nullptr);
+
+	m_BasicPostProcess.reset(new DirectX::BasicPostProcess(RenderingDevice::GetSingleton()->getDevice()));
```

Comment:
isn't `m_BasicPostProcess` part of DirectXTK post-processing effects?

Reply:
fixed